.github/workflows/ci.yml

```diff
@@ -455,3 +455,93 @@ jobs:
           # is merged and the package becomes generally available.
           SELINUX: Permissive
         run: vagrant up --provision-with=selinux,install-runc,test-cri
+
+  integration-macos:
+    name: macOS Integration
+    runs-on: macos-10.15
+    timeout-minutes: 30
+    needs: [project, linters, protos, man]
+
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.13.15'
+
+      - name: Set env
+        shell: bash
+        run: |
+          echo "::set-env name=GOPATH::${{ github.workspace }}"
+          echo "::add-path::${{ github.workspace }}/src/github.com/containerd/containerd/bin"
+
+      - name: Checkout containerd
+        uses: actions/checkout@v2
+        with:
+          path: src/github.com/containerd/containerd
+      - name: Checkout runu
+        uses: actions/checkout@v2
+        with:
+          repository: ukontainer/runu
+          path: src/github.com/ukontainer/runu
+
+      - name: Install Build Deps
+        shell: bash
+        run: |
+          cd src/github.com/containerd/containerd
+          script/setup/install-dev-tools
+      - name: Install runtime
+        shell: bash
+        run: |
+          SHIM_COMMIT=v0.8.2
+          GO111MODULE=on go get -v github.com/ukontainer/runu@"${SHIM_COMMIT}"
+          bindir="$(pwd)/bin/"
+          cd src/github.com/ukontainer/runu
+          git fetch --tags origin "${SHIM_COMMIT}"
+          git checkout "${SHIM_COMMIT}"
```

Comment:
Shouldn't `go get github.com/ukontainer/runu@"${SHIM_COMMIT}"` already pull proper revision?

Reply:
go get with `GO111MODULE=on` doesn't pull source, so we used previously checkouted tree to build the shim binary.
additionally, currently containerd-shim-darwin-v1 can't build with `GO111MODULE=on go get` so, I didn't touch this part.

Reply:
okay; I will use go build to build runu binary.